packages/statemanager/src/merkleStateManager.ts

```diff
@@ -47,6 +48,16 @@ import type {
 import type { Address, DB, PrefixedHexString } from '@ethereumjs/util'
 import type { Debugger } from 'debug'
 
+/**
+ * Prefix to distinguish between a contract deployed with code `0x80`
+ * and `RLP([])` (also having the value `0x80`).
+ *
+ * Otherwise the creation of the code hash for the `0x80` contract
+ * will be the same as the hash of the empty trie which leads to
+ * misbehaviour in the underlying trie library.
+ */
+export const CODEHASH_PREFIX = utf8ToBytes('c')
```

Comment:
This just fixed the state manager browser examples and I thought it was worth the change (no side effects)